src/main/java/com/piekoszek/nowaksharedrent/auth/AuthService.java

```diff
@@ -1,7 +1,9 @@
 package com.piekoszek.nowaksharedrent.auth;
 
+import java.util.Optional;
+
 public interface AuthService {
 
     boolean createAccount(Account account);
```

Comment:
You can make createAccount(Account account); also return token (Optional\<String\>) so that you don't need to login just after create.